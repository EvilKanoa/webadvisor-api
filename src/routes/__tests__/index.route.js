const request = require('supertest');
const app = require('../../app');

describe('Index Route', () => {
  it('displays the correct summary', async () => {
    const response = await request(app)
      .get('/')
      .expect('Content-Type', 'text/html; charset=utf-8')
      .expect(200);
    expect(response.text).toMatchSnapshot();
  });
});
