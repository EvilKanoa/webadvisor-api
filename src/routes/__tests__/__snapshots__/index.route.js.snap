// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.route displays the correct summary 1`] = `"Lightweight and simple GraphQL API for UoG, UW, and WLU."`;
